test/common/halide_test_dirs.h

```diff
@@ -0,0 +1,69 @@
+#ifndef HALIDE_TEST_DIRS_H
+#define HALIDE_TEST_DIRS_H
+
+// This file may be used by AOT tests, so it deliberately does not
+// include Halide.h
+
+#include <string.h>
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <stdio.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#endif
+
+namespace Halide {
+namespace Internal {
```

Comment:
Yeah, mimicking the other file-handlers. Don't feel strongly about it though. Will leave as-is for now.